frontend/src/pages/EditProductPage.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Formik, Form } from 'formik'
+import { Button } from 'react-bootstrap'
+import InputField from '../components/InputField'
+import SelectCategory from '../components/listofproducts/SelectCategory'
+import validationEdit from '../communications/validationEdit'
+import { updateProductData } from '../communications/userApi'
+import EditResponseModal from '../components/modals/EditResponseModal'
+
+const updateOldFieldsInItem = (toBeUpdated, product) => {
```

Comment:
Ezt nem értem mire jó, mert mindig csak egy paraméterrel hívod meg.

Reply:
Mivel nem kell az összes BE-ről érkező mezőt szerkeszteni, illetve az üres mezőknél sem akarunk "" küldeni BE-re, ezért ezeknél az üresen hagyott mezőknél az üres String helyett az eredeti BE adat megy vissza. Erre van jobb megoldás?